packages/core/admin/admin/src/components/Widgets.tsx

```diff
@@ -39,4 +44,184 @@ const ProfileWidget = () => {
   );
 };
 
-export { ProfileWidget };
+/* -------------------------------------------------------------------------------------------------
+ * Key Statistics
+ * -----------------------------------------------------------------------------------------------*/
+const Grid = styled(Box)`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 0;
+  border: 1px solid ${({ theme }) => theme.colors.neutral200};
+  border-radius: ${({ theme }) => theme.borderRadius};
+  overflow: hidden;
+`;
+
+const GridCell = styled(Box)`
+  border-bottom: 1px solid ${({ theme }) => theme.colors.neutral200};
+  border-right: 1px solid ${({ theme }) => theme.colors.neutral200};
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+
+  &:nth-child(2n) {
+    border-right: none;
+  }
+  &:nth-last-child(-n + 2) {
+    border-bottom: none;
+  }
+`;
+
+const formatNumber = ({ locale, number }: { locale: string; number: number }) => {
+  return new Intl.NumberFormat(locale, {
+    notation: 'compact',
+    maximumFractionDigits: 1,
+  }).format(number);
+};
+
+const KeyStatisticsWidget = () => {
+  const { formatMessage, locale } = useIntl();
+  const { data: countDocuments, isLoading: isLoadingCountDocuments } = useGetCountDocumentsQuery();
+  const { data: countKeyStatistics, isLoading: isLoadingKeyStatistics } =
+    useGetKeyStatisticsQuery();
+
+  if (isLoadingKeyStatistics || isLoadingCountDocuments) {
+    return <Widget.Loading />;
+  }
+
+  if (!countKeyStatistics || !countDocuments) {
+    return <Widget.Error />;
+  }
+
+  const mapping: {
+    [key: string]: {
+      label: string;
+      icon: {
+        file: React.ReactNode;
+        background: string;
+        color: string;
+      };
+    };
+  } = {
+    entries: {
+      label: formatMessage({ id: 'widget.key-statistics.list.entries', defaultMessage: 'Entries' }),
```

Comment:
Instead of applying formatMessage here you could instead provide the translation object and only use formatMessage once for label.  Also not too crazy about the vague variable name "mapping"

Reply:
yep! yeah I think something that mentions `keyStatistics` is good 👍